content/docs/command-reference/metrics/diff.md

```diff
@@ -36,19 +36,15 @@ lists all the current metrics, without comparisons.
 
 ## Options
 
-- `--targets <paths>` - limit command scope to these metrics files. Using `-R`,
-  directories to search metrics files in can also be given. When specifying
-  arguments for `--targets` before `revisions`, you should use `--` after this
-  option's arguments, e.g.:
+- `--targets <paths>` - specify the command's scope to these metrics files. It
+  accepts `paths` to any valid metrics file, regardless of whether it's used by
```

Comment:
I think `not used by DVC` is not very clear, can we rephrase it somehow?

Reply:
OK changed to "regardless of whether `dvc.yaml` is currently tracking any params in them". WDYT?

Reply:
it's not even about `dvc.yaml` at this point, even DVC repo might not be initialized. I see that the whole change for that ticket might be out of scope since it's a much bigger change

Reply:
You're right but yes, since iterative/dvc/issues/4446 is still open and the core team hasn't really sent many docs updates per their work on that... We could leave for later.

Reply:
Then again since I just added that note about not needing a DVC project... This text in --targets is kind of contradicting again... I'll simplify it a little for now (less specific).

Reply:
Actually I'm not sure how to generalize since we didn't want to say "not used by DVC". Sry

Reply:
p.s. the current text is not incorrect so... I think it should do for now 🙂